crates/environ/src/compilation.rs

```diff
@@ -186,6 +186,7 @@ pub trait Compiler: Send + Sync {
     fn compile_array_to_wasm_trampoline(
         &self,
         translation: &ModuleTranslation<'_>,
+        tunables: &Tunables,
```

Comment:
One alternative, if you're interested in a little refactoring, would be to store `Tunables` in the `Compiler` itself during its constructor since the tunables are `Engine` invariant

Reply:
Done